docs/resources/cce_node.md

```diff
@@ -399,6 +378,39 @@ extend_param = {
   + `effect` - (Required, String, ForceNew) Available options are NoSchedule, PreferNoSchedule, and NoExecute.
     Changing this parameter will create a new resource.
 
+<a name="extend_params"></a>
+The `extend_params` block supports:
+
+* `max_pods` - (Optional, Int, ForceNew) Specifies the maximum number of instances a node is allowed to create.
+  Changing this parameter will create a new resource.
+
+* `doker_base_size` - (Optional, Int, ForceNew) Specifies the available disk space of a single container on a node,
```

Comment:
the name may be error, how about change to docker_base_size

Reply:
fixed